VisualQnA/README.md

```diff
@@ -55,7 +55,7 @@ Now you can start the frontend UI by following commands:
 ```
 cd ui/
 pip install -r requirements.txt
-python app.py --host 0.0.0.0 --port 7860 --worker-addr http://localhost:8085 --share
+http_proxy= python app.py --host 0.0.0.0 --port 7860 --worker-addr http://localhost:8085 --share
```

Comment:
@Spycsh missing something?

Reply:
No. It's just to unset the http_proxy for this call